cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -1,21 +1,22 @@
 import re
-import pytest
 from random import choice
+import dateparser
 from traceback import format_exc
+from collections import namedtuple
+import pytest
+
 from utils import testgen
 from utils.version import current_version
 from utils.ssh import SSHTail
 from utils.appliance.implementations.ui import navigate_to
-from cfme.web_ui import InfoBlock
-from cfme.configure import tasks
+from cfme.web_ui import paginator
 from cfme.containers.image import Image, list_tbl
 from cfme.containers.provider import ContainersProvider
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import toolbar as tb, flash
-from cfme.configure.tasks import Tasks, tasks_table
-from wait_for import TimedOutError
+from cfme.web_ui import toolbar as tb
```

Comment:
**REQUIRED**
Combine this with your paginator import so everything from web_ui is imported together.